src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -44,13 +47,16 @@ const roleString = {
   READER: 'Reader',
   WRITER: 'Writer',
   OWNER: 'Owner',
-  PROJECT_OWNER: 'Proj. Owner'
+  PROJECT_OWNER: 'Project Owner'
 }
 
-const InfoTile = ({ title, children }) => {
-  return div({ style: Style.dashboard.infoTile }, [
-    div({ style: Style.dashboard.tinyCaps }, [title]),
-    div({ style: { fontSize: 12 } }, [children])
+const InfoRow = ({ title, subtitle, children }) => {
```

Comment:
I did not know about `dl`! Right now you are outputting a `dl` for each row, but I think that the items should all be within the same `dl` per this example: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl
Otherwise it's not actually a list. I also wonder about the role of "row"; how did you decide on that?

BTW, when using the screenreader the elements within the collapse ALL get read out when the button is in the expanded state. I think that is because of the aria "owns" attribute. It seems excessively verbose to me, and I can't figure out how to actually navigate into the div contents to read the individually. But that is all within the existing Collapse element, so if there's an issue to be fixed there I don't expect it to be part of this PR.

Reply:
I'm not actually sure how I landed on the row role, it might just be an artifact of some old code that I scrapped. 

Agreed that the screenreader readout is a bit much, I can file a followup ticket to look into improving the behavior of that within a Collapse element

Reply:
Filed as https://broadworkbench.atlassian.net/browse/SU-92